.github/workflows/ci.yaml

```diff
@@ -0,0 +1,43 @@
+name: Node.js CI
+
+on:
+  schedule:
+    - cron:  '23 4 * * *' # Once, nightly
```

Comment:
I think github started disabling these sort of nightly runs.  What value do you see in them when we also run on PR and pushes to `main`?

Reply:
We are using this setup in `wiby` and I assumed this can be also applied here.

Reply:
I would love to hear the reasoning behind the decision.  In projects which are not active daily from multiple contributors I do not usually see the point in a nightly build.  I guess I just think of a nightly as a way to integration test many in-flight features from different engineers, so without that it seems better to just test each commit on a PR branch and `main`.

Reply:
I removed it for now and maybe we can kindly summon @dominykas here to add more thoughts regarding night scheduler.